database/db_connection.js

```diff
@@ -0,0 +1,24 @@
+const { Pool } = require('pg');
+const url = require('url');
+
+const env = require('env2');
+env('./config.env');
+
+if(!process.env.GROW_DB_URL) throw new Error('Environment variable GROW_DB_URL must be set');
```

Comment:
Easier to read diffs if you try to keep lines < 80 chars

![image](https://cloud.githubusercontent.com/assets/12845233/23900091/bc6e5b24-08af-11e7-9e17-1702af139ee4.png)